src/player.cpp

```diff
@@ -4160,14 +4160,14 @@ void player::pause()
     search_surroundings();
 }
 
-void player::shout( std::string msg )
+void player::shout( std::string msg, const std::string &verbal_shout_id )
 {
-    int base = 10;
+    int base = 85;
```

Comment:
Old ones were OK for screams. Calling should have a shorter range (since it needs to be understood) - for example reducing the `shout_multiplier` to 1 when calling.